app/uk/gov/hmrc/gform/auth/AuthService.scala

```diff
@@ -67,6 +72,29 @@ class AuthService(
         performAgent(agentAccess, formTemplate, lang, ggAuthorised(RecoverAuthResult.noop), ifSuccessPerformEnrolment)
     }
 
+  private def performAWSALBAuth(formTemplate: FormTemplate, lang: Option[String])(
```

Comment:
Neither of `formTemplate` and `lang` parameters are used.

Reply:
Yep, will remove those.